### PR TITLE
Add documentation about the prerequisites for pennyworth subjects hosts

### PR DESCRIPTION
**Pull request by aduffeck**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 02afa9c75c396895f7e7f27628d481f4b089e50d into 75842604805b17c8311575620c49de84111c6f11 )

File: README.md [hunk 1]
```diff
@@ -236,6 +236,24 @@ hosts:
...
+  3. There can't be any subvolumes below `/` besides `.snapshots`
+  4. It's usually helpful to exclude `*/.ssh` from the rollback so that SSH
+     access is retained
+
+       echo "*/.ssh" > /etc/snapper/filters/ssh.txt
+
+  5. There needs to be a snapper snapshot of the defined state which will be
+     configured in the `hosts.yml`:
+
+       snapper create --description "Initial snapshot"
```
Comment (cornelius) [inline README.md]: It might be nice to add a hint where to get the id from which has to be put into the `hosts.yaml`.

File: README.md [hunk 2]
```diff
@@ -236,6 +236,24 @@ hosts:
...
+       snapper create-config /
+
+  3. There can't be any subvolumes below `/` besides `.snapshots`
+  4. It's usually helpful to exclude `*/.ssh` from the rollback so that SSH
+     access is retained
+
+       echo "*/.ssh" > /etc/snapper/filters/ssh.txt
+
+  5. There needs to be a snapper snapshot of the defined state which will be
+     configured in the `hosts.yml`:
```
Comment (cornelius) [inline README.md]: The suffix we use is `yaml` not `yml`

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 7f57597c275fd42b5ca3b916a5870aef04f5b84f into 75842604805b17c8311575620c49de84111c6f11 )

**Comment by cornelius** [comment]
lgtm

**Comment by greygoo** [comment]
lgtm

**Pull request merged**